cmd/boulder-remoteva/main.go

```diff
@@ -0,0 +1,126 @@
+package notmain
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/bdns"
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/features"
+	bgrpc "github.com/letsencrypt/boulder/grpc"
+	"github.com/letsencrypt/boulder/va"
+	vacfg "github.com/letsencrypt/boulder/va/config"
+	vapb "github.com/letsencrypt/boulder/va/proto"
+)
+
+type Config struct {
+	RVA struct {
+		vacfg.Shared
+	}
+
+	Syslog        cmd.SyslogConfig
+	OpenTelemetry cmd.OpenTelemetryConfig
+}
+
+func main() {
```

Comment:
I thought we'd discussed maybe having a way to factor out most of the `main()` so that we're not copy-pasting code between boulder-va/main.go and boulder-rva/main.go. Did we change our mind on that, or did we run into a technical reason why that doesn't work well?